ophyd/pseudopos.py

```diff
@@ -126,11 +334,14 @@ class PseudoPositioner(Device, Positioner):
     '''
     def __init__(self, prefix, *, concurrent=True, read_attrs=None,
                  configuration_attrs=None, monitor_attrs=None, name=None,
-                 **kwargs):
+                 egu='', **kwargs):
```

Comment:
Maybe where 'mm, mm, mm, mm' might be annoying to see (it isn't a fast-food commercial after all).
I think you'd probably want to be able to specify that the pseudopositioner units are just 'mm' all around